apps/pollbook/backend/src/voter_checklist.tsx

```diff
@@ -106,6 +108,7 @@ const VoterTable = styled.table`
     border-bottom: 1px solid black;
     text-align: left;
     vertical-align: top;
+    white-space: nowrap;
```

Comment:
Without this the `Voter ID` header wraps to 2 lines when other columns are very wide due to long names or addresses.